### PR TITLE
Changed default style to match canada.ca default button background co…

### PR DESCRIPTION
**Pull request by Maxim-Lam**
…lor and applied it to modal cancel button.

### Description

List of proposed changes:

Changed the background color of the default styling to match canada.ca one.

Applied the style to the modal's "no" button

Also fix the focus issue bug

### What to test for/How to test

Visit status page, look at cancel button and put it in focus by either tabbing to it or hold left click on it. Text should be visible now.

Select cancel and do the same thing with the no button

### CODE REVIEW

File: components/ActionButton.tsx [hunk 1]
```diff
@@ -20,11 +20,11 @@ const ActionButton: FC<ActionButtonProps> = ({
   switch (style) {
     case 'primary':
       classStyle +=
-        'border-blue-dark bg-blue-dark text-basic-white focus:bg-blue-normal hover:bg-blue-normal active:bg-blue-active'
+        'border-blue-dark bg-blue-dark text-basic-white focus:bg-blue-normal hover:bg-blue-normal active:bg-blue-active shadow-sm'
       break
     default:
       classStyle +=
-        'border-gray-default bg-gray-light text-blue-light hover:border-gray-deep hover:bg-gray-dark'
+        'border-gray-dark bg-gray-normal text-blue-light hover:border-gray-deep hover:bg-gray-dark focus:text-blue-light focus:bg-gray-dark shadow-sm'
```
Comment (krischarbonneau) [inline components/ActionButton.tsx:27]: Going off the design pattern for the secondary button [here](https://design.canada.ca/common-design-patterns/buttons.html) we can see that the button has an outset border. There isn't an outset border style in tailwind but we can simulate that by setting borders on individual sides like the below suggestion.

```suggestion
        'border-gray-dark bg-gray-normal text-blue-light hover:bg-gray-dark hover:border-l-gray-deep hover:border-t-gray-deep focus:text-blue-light focus:bg-gray-dark shadow-sm border-r-gray-500 border-b-gray-500'
```

Comment (Maxim-Lam) [reply components/ActionButton.tsx:27]: very nice. I was bummed that there wasn't a way to do it like 'border-r-b...'. good suggestion

Comment (krischarbonneau) [reply components/ActionButton.tsx]: Tailwind is a great tool that makes styling easy, but it still can't hold a candle to what you can do with conventional CSS

**Review by krischarbonneau — APPROVED** [review]
Looks good!

File: components/ActionButton.tsx (hunk 1 above)
Comment (Maxim-Lam) [reply components/ActionButton.tsx]: omg i agree

**Pull request merged**